packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1290,19 +1295,17 @@ export default class Meetings extends WebexPlugin {
 
       if (meetingInfo) {
         meeting.injectMeetingInfo(meetingInfo, meetingInfoOptions, meetingLookupUrl);
-      } else if (
-        enableUnifiedMeetings &&
-        !isMeetingActive &&
-        useRandomDelayForInfo &&
-        waitingTime > 0
-      ) {
-        meeting.fetchMeetingInfoTimeoutId = setTimeout(
-          () => meeting.fetchMeetingInfo(meetingInfoOptions),
-          waitingTime
-        );
-        meeting.parseMeetingInfo(undefined, destination);
-      } else {
-        await meeting.fetchMeetingInfo(meetingInfoOptions);
+      } else if (!isOne2OneMeeting) {
```

Comment:
Just thinking about it, would it make more sense to use the existing `type` parameter in `meetings.create` and then not do anything related to `meetingInfo` if the type is the newly defined one?

Also, `isOne2OneMeeting` doesn't feel like a standard one.

Reply:
yes. removed isOne2OneMeeting and utilised type